apps/gradient_autoscheduler/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+add_library(gradient_autoscheduler
+            SHARED
+            GradientAutoscheduler.cpp
+            ASLog.cpp)
+target_link_libraries(gradient_autoscheduler PRIVATE "${HALIDE_COMPILER_LIB}")
+target_include_directories(gradient_autoscheduler PRIVATE "${HALIDE_INCLUDE_DIR}" "${HALIDE_TOOLS_DIR}")
+install(TARGETS gradient_autoscheduler LIBRARY DESTINATION bin)
```

Comment:
You need to add these lines before `install(...)`:

```
if(NOT MSVC)
  set_target_properties(gradient_autoscheduler PROPERTIES LINK_FLAGS "-rdynamic")
elseif(APPLE)
  set_target_properties(gradient_autoscheduler PROPERTIES LINK_FLAGS "-undefined dynamic_lookup")
endif()

if(NOT HALIDE_ENABLE_RTTI)
  if(NOT MSVC)
    target_compile_options(gradient_autoscheduler PRIVATE "-fno-rtti")
  else()
    target_compile_options(gradient_autoscheduler PRIVATE "/GR-")
  endif()
endif()
```

Reply:
Okay.  I will add this to the change.

Reply:
Cherry-picked your e28ccd87e57c174a1c9afd9413e5d816400f84a3 with said change.